test/Tester/StreamingTests/ProgrammaticSubscribeTests/SubscriptionObserverWithImplicitSubscribingTestRunner.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans;
+using Orleans.TestingHost.Utils;
+using TestExtensions;
+using UnitTests.GrainInterfaces;
+using UnitTests.Grains;
+using UnitTests.Grains.ProgrammaticSubscribe;
+using Xunit;
+
+namespace Tester.StreamingTests.ProgrammaticSubscribeTests
+{
+    public class SubscriptionObserverWithImplicitSubscribingTestRunner: OrleansTestingBase
+    {
+        private static readonly TimeSpan _timeout = TimeSpan.FromSeconds(30);
+        //test utilities and statics
+        public static string StreamProviderName = "StreamProvider1";
+        public static string StreamProviderName2 = "StreamProvider2";
+        private BaseTestClusterFixture fixture;
+        public SubscriptionObserverWithImplicitSubscribingTestRunner(BaseTestClusterFixture fixture)
+        {
+            this.fixture = fixture;
+        }
+
+        [SkippableFact]
+        public async Task StreamingTests_ImplicitSubscribProvider_DontHaveSubscriptionManager()
```

Comment:
This test is true for implicit-only.  Default supports implicit and explicit, in which case this should fail, correct?  Should this be explicit in the test name?